<launch>
  <rosparam ns="base/tready_node" command="load" file="$(find hebi_cpp_api_examples)/config/tready_base_params.yaml"/>

  <!-- Robot components -->
  <node ns="base" pkg="hebi_cpp_api_examples" type="tready_node.py" name="tready_node" output="screen" required="true"/>
  
</launch>
